internal/storage/storageutil/client_helper.go

```diff
@@ -0,0 +1,111 @@
+package storageutil
```

Comment:
nit: given that client_helper.go is already in a package/directory that has util in its name i.e. storage"util". So do you also think we can simply call this file "client.go" ?